.gitignore

```diff
@@ -75,7 +75,11 @@ Thumbs.db
 
 *.h2.db
 *.trace.db
-/components/event/event-listener/conf/groups.properties
-/components/event/event-listener/conf/login.config
-/components/event/event-listener/conf/openejb.xml
-/components/event/event-listener/conf/users.properties
\ No newline at end of file
+
+dependency-reduced-pom.xml
+src/test/resources/configunit.yaml
```

Comment:
Removed all the git ignores as all these files are now moved to external file location during test